.buildkite/pipeline.yml

```diff
@@ -10,20 +10,6 @@ steps:
       DEVELOPER_DIR: "/Applications/Xcode13.app"
     command: scripts/license_finder.sh
 
-  - label: ':docker: Build expo maze runner image'
-    key: "expo-maze-runner-image"
-    timeout_in_minutes: 10
-    plugins:
-      - docker-compose#v3.9.0:
-          build: expo-maze-runner
-          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
-          cache-from:
-            - expo-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-${BRANCH_NAME}
-      - docker-compose#v3.9.0:
-          push:
-            - expo-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-${BRANCH_NAME}
-            - expo-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-latest
-
```

Comment:
what was this doing previously?

Reply:
It was building a Docker image based on the Maze Runner image, adding an extra layer to copy in the `features` directory.  That's completely unnecessary as we can just use the pre-built Maze Runner image directly and mount a volume for access to the features directory.